src/meta/css.js

```diff
@@ -139,6 +140,20 @@ function getFontawesomeStyle() {
 	return styles.map(style => `@import "fontawesome/style-${style}";`).join('\n');
 }
 
+async function copyFontAwesomeFiles() {
+	await mkdirp(path.join(__dirname, '../../build/public/fontawesome/webfonts'));
+	const fonts = await fs.promises.opendir(path.join(utils.getFontawesomePath(), '/webfonts'));
+	const copyOperations = [];
+	for await (const file of fonts) {
```

Comment:
Do you need this `await` here? Seems like it can removed 🤔

Reply:
You do - `fs.Dir` (from `fs/promises`) is an async iterator. Omitting the `await` results in `TypeError: fonts is not iterable`

Reply:
@oplik0 TIL 😆